### PR TITLE
Bump to 0.9.0

### PR DESCRIPTION
**Pull request by chartgerink**
* Bumps specs to 0.9.0 with some minor changes
* Adds the files in their respective `specs/` folder for website accessibility.

Note that the URLs only work when we sync the `gh-pages` branch with `main` after merge.

[https://p2pcommons.com/specs/module/1.0.0](https://p2pcommons.com/specs/module/1.0.0)

### CODE REVIEW

File: module.md [hunk 1]
```diff
@@ -275,7 +263,7 @@ The links used in the examples do not work and serve illustrative purposes only.
   "url": "hyper://cca6eb69a3ad6104ca31b9fee7832d74068db16ef2169eaaab5b48096e128342",
   "links": {
      "license": [{"href": "https://creativecommons.org/publicdomain/zero/1.0/legalcode"}],
-     "spec": [{"href": "https://p2pcommons.com/specs/module/0.0.0.html"}]
+     "spec": [{"href": "https://p2pcommons.com/specs/module/x.x.x."}]
```
Comment (jameslibscie) [inline module.md:266]:
```suggestion
     "spec": [{"href": "https://p2pcommons.com/specs/module/x.x.x"}]
```

**Comment by chartgerink** [comment]
Ah thanks, good catch!

The `v0.9.0` is to indicate we're closing in on `v1.0.0` but I would like to give it just a bit more of a trial run before we bump it to `v1.0.0`. We might still find an issue or two that we'd like to update. This indicates it's not yet stable.

**Comment by juliangruber** [comment]
> The `v0.9.0` is to indicate we're closing in on `v1.0.0` but I would like to give it just a bit more of a trial run before we bump it to `v1.0.0`.

Just so I understand correctly: You wouldn't have an issue with a `v0.10.0` or bumping from a `v0.5.0` to `v1.0.0`, in general, right?

**Comment by chartgerink** [comment]
Correct :)

**Pull request merged**